### PR TITLE
Merge Tiegan working branch into main

### PR DESCRIPTION
**Pull request by tieganh**
Adding Georgia Strait and Val des Bois scenarios, along with other minor changes to scripts and workflows. Still need to resolve conflicts and then make snapshots (md files) for new scenarios.

### CODE REVIEW

**Comment by tieganh** [comment]
I want to keep the new (GeorgiaStraitScenario branch) version of the conflicting file (FINISHED/ACM7p3_LeechRiverFullFault.md), but don't know how to do so safely. Can @DamonU2 @anthonyfok help?

File: scripts/plotRupture.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+from cartopy.mpl.gridliner import LONGITUDE_FORMATTER, LATITUDE_FORMATTER
+import numpy as np
+
+### Load file
+NAME = sys.argv[1]
+rup = minidom.parse(fileloc+NAME)
+hypo = rup.getElementsByTagName('hypocenter')
+lat = hypo[0].attributes['lat'].value
+lon = hypo[0].attributes['lon'].value;
+
```
Comment (DamonU2) [inline scripts/plotRupture.py:31]: Unnecessary semicolon

File: scripts/weightedAverage.py [hunk 2]
```diff
@@ -51,7 +51,7 @@
     lastcol = 'lat'
     groupname = 'asset_id'
 elif runType.lower() == 'consequence':
-    name_native = "consequences-rlz"; name_native_long = "consequences-rlz"
+    name_native = "consequences"; name_native_long = "consequences-rlz"
```
Comment (DamonU2) [inline scripts/weightedAverage.py:54]: Unnecessary semicolon in line 112 (I can only comment here, unfortunately)

**Review by DamonU2 — APPROVED** [review]

**Comment by tieganh** [comment]
Fixed conflict, Damon's revisions and the Leech River recurrence in [new] TakeSnapshot.py

**Pull request merged**